src/pages/PageNotFound.tsx

```
@@ -1 +1,20 @@
-export const PageNotFound = () => <div>PageNotFound</div>;
+import { useNavigate } from 'react-router-dom';
+
+import { Error } from '@components';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)